### PR TITLE
Create Volunteer Plugin Doc Writers

### PR DESCRIPTION
**Pull request by AB1908**
## Added
A volunteer docs page as discussed on Discord: https://discord.com/channels/686053708261228577/840286238928797736/936621649829199923

## Checklist
- [X] before creating a new note, I searched the vault
- [X] new notes have the `.md` extension
- [X] (if applicable) attached images have descriptive file names
- [X] (if applicable) for new notes in the folder "04 - Guides, Workflows, & Courses", I added a link to them in one of the "for {group X}" overviews: https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

### CODE REVIEW

**Comment by claremacrae** [comment]
A note to reviewers:
- I wonder whether the title "Volunteer **Plugin** Doc Writers" might be clearer?
- I don't know if it matters that there's no metadata at the top of the file...

**Review by claremacrae — CHANGES_REQUESTED** [review]
The checks have pointed out that there is no .md file extension on this file...

If you know how, please could you rename it to fix that?

Many thanks

**Comment by AB1908** [comment]
I dunno how I missed something like that. I made a few of the changes but have left if with no metadata.

**Review by argenos — APPROVED** [review]

**Comment by argenos** [comment]
@claremacrae I believe you need to approve since you requested changes last time

**Review by claremacrae — APPROVED** [review]
@argenos I'll approve this, because I cannot see how to now apply the suggested edit that I made previously, which was to fix a typo:
You can reach **out** the the plugin developer
->
You can reach out to the plugin developer

**Comment by claremacrae** [comment]
Thanks for fixing the file extension! :-)

File: 06 - Inbox/Volunteer Plugin Doc Writers.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Volunteer Plugin Doc Writers
+
+Half (maybe all?) the fun of using Obsidian is its comprehensive set of plugins. We recently crossed the 500 mark! However, a few plugins could use some better documentation. We'd like to help bridge that gap.
+
+We'll maintain a list of plugins that need better docs. You can reach out the the plugin developer and collaborate with them to help out. You don't necessarily need technical skills to contribute; you could add "Getting Started" pages to bring users up to speed about how to use the plugin. That said, some technical ability will be of help too.
```
[the remark this answers is not in the document]
Comment (AB1908) [reply 06 - Inbox/Volunteer Plugin Doc Writers.md:5]: Oh my. These mistakes keep slipping through. I'm so sorry ahaha.

**Comment by claremacrae** [comment]
Merged - thank you very much @AB1908

**Pull request merged**